src/odo.ts

```diff
@@ -93,7 +93,7 @@ export const Command = {
     linkComponentTo: (project: string, app: string, component: string, componentToLink: string, port?: string) =>
         `odo project set ${project} && odo application set ${app} && odo component set ${component} && odo link ${componentToLink} --wait${port ? ' --port ' + port : ''}`,
     pushComponent: (project: string, app: string, component: string) =>
-        `odo push ${component} --app ${app} --project ${project}`,
+        `odo push ${component} --app ${app} --project ${project} -v 9`,
```

Comment:
It should be configurable not just hard coded. 
Options:
* configuration settings
   - verbose output level - could be integer 0 - no verbose output, > 0 - verbosity level
* command `OpenShift: Set log verbosity level` that would request verbosity level from user and write it to settings (if that possible)
* after first `OpenShift: Push` ask if dev want to increase verbosity level with options: Yes, No, Do not show again.  Yes would call command above.

Reply:
@dgolovin there should be three option `Yes, No, Do not show again` or just `Yes, No` option?. what if I want to change the verbose level. I think we can skip the `Do not show again` part. Every time when the user uses the `push` command we will show the information message that he/she wants to change the verbose level.